### ISSUE TITLE
Syntax highlighting break after function call in include

### ISSUE THREAD

**Issue by JosephSilber** [Medalink/laravel-blade#72]
This works well:

```
@include('products.partials.add-to-cart', [
	'quantity' => 1,
	'price' => $product->present()->price,
])
```

![highlighting works](http://i.imgur.com/2KeTzsl.png)

But this breaks after the parentheses:

```
@include('products.partials.add-to-cart', [
	'price' => $product->present()->price,
	'quantity' => 1,
])
```

![highlighting doesn't work](http://i.imgur.com/yZ5QlAZ.png)

Here's a video of it in action: [sublime-blade.mov](https://dl.dropboxusercontent.com/u/6484839/sublime-blade.mov)

---

Might be related to #67

**Comment by wmeredith** [comment]
I have this same problem.
![screen shot 2015-07-07 at 2 39 05 pm](https://cloud.githubusercontent.com/assets/639184/8555493/145dda56-24b6-11e5-826d-b76f3c6ca5fe.png)

**Comment by Medalink** [comment]
Fixed in 711179b7dc07824f218dee7855f38a0829672efc.
